packages/tinyemu/tinyemu_patch.patch

```diff
@@ -0,0 +1,121 @@
+diff -uNr tinyemu-2019-12-21/Makefile tinyemu-2019-12-21.mod/Makefile
+--- tinyemu-2019-12-21/Makefile	2019-12-22 00:23:54.000000000 +0700
++++ tinyemu-2019-12-21.mod/Makefile	2022-04-04 17:35:13.619945792 +0700
+@@ -29,7 +29,6 @@
+ CONFIG_SDL=y
+ # if set, compile the 128 bit emulator. Note: the 128 bit target does
+ # not compile if gcc does not support the int128 type (32 bit hosts).
+-CONFIG_INT128=y
```

Comment:
It would make sense to do this only for 32 bit archs.

Rename the current patch file to 002-Makefile.patch and split other hard-coded paths fix to 003-fix-hardcoded-paths.patch, and create a 002-Makefile.patch32 which removes this option. This will make sure that patches are applied in the correct order.

Reply:
I see, Will do it asap